### DISCUSSION TITLE
Favorite emotes per channel

### DISCUSSION THREAD

**Discussion by lastmetroid** [night/betterttv#6381] [Ideas]
Would be nice if you could add a optional GUI in the emote menu, where users can add unlimited favorite icons for each channel, in the same style of frequently used, but user defined. maybe add a simple way of adding and removing them too by  ctrl+click or something to add and shift+click to remove

**Comment by dclstn** [comment]
This exists, try alt-clicking an emote in the emote-menu and it'll become pinned at the top.

**Comment by lastmetroid** [comment]
Oh nice. Would it maybe be possible to add a function to this so that you can move the position of the favorite emotes? click on it and then drag them around so you can sort them in order. Also would be nice if alt +click works inside the chat window to add them not just the emote search gui @dclstn
